### PR TITLE
Sldss 27892 tqc ref do not work properly

### PR DESCRIPTION
**Pull request by achebanets**
The problem is: we use latest version of ReDoc. Latest version doesn't support the directive $ref the second level (we use it for TQC endpoint description). So the script resolve the $ref and replace with the certain value.

### CODE REVIEW

File: spec/process-yaml.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const yaml = require('js-yaml');
+const fs = require('fs');
+const { Resolver }  = require('@stoplight/json-ref-resolver');
```
[the remark this answers is not in the document]
Comment (achebanets) [reply spec/process-yaml.js:3]: Removed

File: spec/process-yaml.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            }
+        });
+        console.log("-----------------------------------------------------------");
+        console.log("before resolver.resolve(..)");
+        const resolved = await resolver.resolve(doc, { jsonPointer: "#/x-paths" });
+        console.log("after resolver.resolve(..)");
+        console.log("-----------------------------------------------------------");
+        console.log(resolved);
+        console.log("before resolved.result");
+        var destinationJson = {"x-paths": resolved.result}
```
[the remark this answers is not in the document]
Comment (achebanets) [reply spec/process-yaml.js:34]: changed

**Comment by DmitryBochorishvili** [comment]
Could you describe what's the problem you're solving and what's the overall solution?
It's hard to get it from the changes itself

**Comment by DmitryBochorishvili** [comment]
Couple of questions:
* should we add spec/.DS_Store to gitignore?
* are there any changes required to travis build script for the project?

**Comment by achebanets** [comment]
> Couple of questions:
> 
> * should we add spec/.DS_Store to gitignore?
> * are there any changes required to travis build script for the project?

**Review by DmitryBochorishvili — APPROVED** [review]

**Review by alubenskyi — APPROVED** [review]

**Review by AlexArcher — APPROVED** [review]

**Review by rischenko — APPROVED** [review]

**Pull request merged**